sklearn/model_selection/_split.py

```diff
@@ -49,6 +49,17 @@
            'check_cv']
 
 
+NSPLIT_WARNING = (
+    "You should specify a value for 'n_splits' instead of relying on the "
+    "default value. Note that this default value of 3 is deprecated in "
```

Comment:
Instead of "Note...", I would say "This default value will change from 3 to 5 in version 0.22."